packages/client/hmi-client/src/components/documents/tera-document-asset.vue

```diff
@@ -308,6 +310,25 @@ onUpdated(() => {
 		emit('asset-loaded');
 	}
 });
+
+onMounted(async () => {
+	await subscribe(ClientEventType.Extraction, subscribeToExtraction);
```

Comment:
You may want to listen for `ExtractionPdf` if you are looking for pdf extraction to be done.